examples/redux/README.md

```diff
@@ -0,0 +1,41 @@
+# Usage with Redux
+
+#### 1. Add dependencies
+``` bash
+npm install airbrake-js redux-airbrake --save
+```
+
+#### 2. Import dependency
+``` js
+import airbrakeMiddleware from 'redux-airbrake';
+```
+
+#### 3. Configure & add middleware
+``` js
+const errorMiddleware = airbrakeMiddleware({
+    projectId: '******',
+    projectKey: '**************'
+});
```

Comment:
So what about accepting airbrake-js Client instance? Do you see any drawbacks?

>It already supports configuration for filters, etc.

For example, how can I add a filter?